src/plugins/customBorders/customBorders.js

```diff
@@ -188,24 +186,14 @@ class CustomBorders extends BasePlugin {
     * @param {Array} selection
     */
   getBorders(selection) {
+    if (!selection) {
```

Comment:
Maybe safer will be using `!Array.isArray(selection)` expression?

Reply:
Yes, of course. Thanks.